gnssanalysis/gn_diffaux.py

```diff
@@ -547,7 +549,7 @@ def diffsp3(
         hlm_mode=hlm_mode,
         plot=plot,
         write_rac_file=write_rac_file,
-    )
+    )  # Eugene: sisre() returns SISRE instead of RAC differences
```

Comment:
comment to remove. this is in the description of the sisre function

Reply:
Gone, changed the variable names which were confusing